extensions/amp-social-share/0.1/amp-social-share.css

```diff
@@ -19,51 +19,44 @@
 amp-social-share {
```

Comment:
No longer needed here, right? I mean it's fine to keep in general, but I'd be afraid of value skew which would cause a sudden resize when this CSS is loaded.